### PR TITLE
Responsive styles

### PR DESCRIPTION
**Pull request by rohanshiva**
Sectioned out all the major sections like Navbar, Left Leaderboard, Editor, etc by adding `max/min-width`, `max/min-height` to all these sections in terms of the viewport. Therefore, it will be more responsive and it should look similar on all devices. Still need to look into making the font responsive, as everything is currently using fixed units (rem). However, it shouldn't be too bad, as we most likely need to set the `font-size` attribute for the HTML element for different sizes using media queries tags.

### CODE REVIEW

File: client/src/components/game/Game.tsx [hunk 1]
```diff
@@ -27,6 +27,8 @@ import useSocket from "./hooks/socket/UseSocket";
 import IPrompt from "../../interfaces/Prompt";
 import IAnswer from "../../interfaces/Answer";
 import Timer from "../timer/Timer";
+import Choices from "../choices/Choices";
+import { stat } from "fs";
```
Comment (Ramko9999) [inline client/src/components/game/Game.tsx]: might have accidentally imported this

File: client/src/components/game/Game.css [hunk 2]
```diff
@@ -49,44 +54,36 @@
...
   font-weight: 600;
   background-color: var(--primary);
   color: var(--bg);
   outline: none;
   border: none;
+  height: min-content;
 }
 
-button:hover {
+.game-buttons:hover {
```
Comment (Ramko9999) [inline client/src/components/game/Game.css:75]: might be a good idea to add the cursor pointer over these buttons

**Comment by Ramko9999** [comment]
> Sectioned out all the major sections like Navbar, Left Leaderboard, Editor, etc by adding `max/min-width`, `max/min-height` to all these sections in terms of the viewport. Therefore, it will be more responsive and it should look similar on all devices. Still need to look into making the font responsive, as everything is currently using fixed units (rem). However, it shouldn't be too bad, as we most likely need to set the `font-size` attribute for the HTML element for different sizes using media queries tags.

File: client/src/components/choices/Choices.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  choices: string[];
+  guessHandler: (guess: string) => void;
+}
+
+function Choices({ choices, guessHandler = (guess: string) => {} }: IChoices) {
+  const [selected, setSelected] = useState<string>("");
+  const [hasGuessed, setHasGuessed] = useState<boolean>(false);
+
+  const confirmGuessHandler = (event: any) => {
+    guessHandler(selected);
```
Comment (Ramko9999) [inline client/src/components/choices/Choices.tsx]: seems like you can confirm without actually picking an option

File: client/src/components/game/Game.tsx [hunk 4]
```diff
@@ -53,141 +55,168 @@ const initialState: IGameState = {
...
+        </div>
+
+        {inLobby() && <Editor chunk={lobbyChunk} />}
+        {inGuessing() && (
+          <>
+            <Editor chunk={(state.prompt as IPrompt).chunk} />
+          </>
+        )}
+        {inDoneGuessing() && (
+          <Answer
```
Comment (Ramko9999) [inline client/src/components/game/Game.tsx:194]: Are you going to show the repository after they guessed? Will this be in a separate commit?

Comment (rohanshiva) [reply client/src/components/game/Game.tsx:194]: Separate pr

File: client/src/components/repoCard/RepoCard.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+          <Icon.Book className="icon" />
+          {repository.name}
+        </div>
+        <div className="repo-desc">
+          {repository.description
+            ? repository.description
+            : "No description provided"}
+        </div>
+        <div className="repo-stats">
+          <div className="repo-language">{repository.language}</div>
```
Comment (Ramko9999) [inline client/src/components/repoCard/RepoCard.tsx:26]: repository.language may be null
